### DISCUSSION TITLE
Multi-columns item height

### DISCUSSION THREAD

**Discussion by sknr15** [Shopify/flash-list#630] [Q&A]
### **Problem:**
If there are multiple columns and items with different length of content, the items within a row don't have the same height.

To show you the exact problem, I wrote a small code snippet which shows items of different heights.

[Expo snack](https://snack.expo.dev/oaOy4E5hA)

I would like to give every item within a row the same height.
Using the biggest possible item height is not an option as there are rows with only small items.
For example: in this code snippet the first row would look strange if every first-row item has the height of item 'jkl'.

Is there any thing I'm missing here or is there a workaround for this?

Appreciate any help on this one!

**Comment by gerwim** [comment]
@sknr15 Did you ever find a solution to this?
